itest/containers/containers.go

```diff
@@ -0,0 +1,186 @@
+package containers
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"regexp"
+	"testing"
+	"time"
+
+	"github.com/ory/dockertest/v3"
+	"github.com/ory/dockertest/v3/docker"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	bitcoindContainerName = "bitcoind-test"
+)
+
+var errRegex = regexp.MustCompile(`(E|e)rror`)
+
+// Manager is a wrapper around all Docker instances, and the Docker API.
+// It provides utilities to run and interact with all Docker containers used within e2e testing.
+type Manager struct {
+	ImageConfig
```

Comment:
```suggestion
	cfg *ImageConfig
```
Probably the config does not need to be exposed